### PR TITLE
Fix warning when printing level of ">"

### PR DESCRIPTION
**Pull request by lzutao**

### CODE REVIEW

File: libr/anal/meta.c [hunk 1]
```diff
@@ -234,8 +234,6 @@ static void r_meta_del_cb (RAnal *a, int type, int rad, SdbForeachCallback cb, v
 		a->cb_printf ("[");
 	}
 
-	RAnalMetaUserItem ui = { a, type, rad, cb, user, 0, NULL };
-
```
Comment (radare) [inline libr/anal/meta.c]: Unrelated?

Comment (lzutao) [reply libr/anal/meta.c]: Yes. Unrelated but it is unused var.
